client/src/components/Buttons.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faThumbsUp, faThumbsDown } from "@fortawesome/free-solid-svg-icons";
+
+const Buttons = () => {
+  const [totalVotes, setTotalVotes] = useState(0);
+  const increaseVote = () => {
+    setTotalVotes(totalVotes + 1);
+  };
+  const decreaseVote = () => {
+    if (totalVotes >= 0) {
+      setTotalVotes(totalVotes - 1);
```

Comment:
at the moment the decrease is allowing votes to go to -1, rather than stopping at 0 (using > rather than >= might be a good idea)